packages/test/test-end-to-end-tests/.vscode/launch.json

```diff
@@ -0,0 +1,235 @@
+				"fluid__test__backCompat": "${input:backCompat}",
+				"FLUID_TEST_VERBOSE": "${input:verboseConsoleOutputt}",
+			},
+			"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/mocha",
+			"windows": {
+				"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/mocha.cmd",
+			},
+			"sourceMaps": true,
+			"program": "${file}",
+			"args": [
+				// "--fgrep",               // Uncomment to filter by test case name
+				// "<test case name>",
+				"--no-timeouts",
+				"--exit",
+			],
+			"cwd": "${fileDirname}",
+			"skipFiles": ["<node_internals>/**", "**/node_modules/!(.legacy)/**"],
+			"outFiles": [
+				"${workspaceFolder}/lib/**/*.js",
+				"${workspaceFolder:FluidFramework}/**/{dist,lib}/**/!(*.spec,*.test,*.tests).[mc]?js",
+				"${input:debugOldFluidModules}",
+			],
+			"presentation": {
+				"group": "e2e-tests",
+				"order": 4,
+			},
+			"preLaunchTask": "Build Current Tests",
+			"internalConsoleOptions": "openOnSessionStart",
+		},
+	],
+	"compounds": [
+		{
+			"name": "Debug Current e2e test (tinylicious)",
+			"configurations": ["tinylicious", "Client: Debug Current e2e test (tinylicious)"],
+			"presentation": {
+				"group": "e2e-tests",
+				"order": 2,
+			},
+		},
+	],
+	// Note: if launching some debug configuration with non-default values repeatedly, you can use "restart" to avoid specifying
+	// the inputs each time.
+	// We could consider using something like https://marketplace.visualstudio.com/items?itemName=spadin.memento-inputs to make
+	// this a bit more ergonomic, but that would require extension installation to make these launch targets work.
+	"inputs": [
+		{
+			"id": "argDriver",
+			"description": "e2e test driver to use",
+			"default": "local",
+			"type": "pickString",
+			"options": ["local", "tinylicious", "routerlicious", "odsp"],
+		},
+		{
+			"id": "backCompat",
+			"description": "compatibility matrix",
+			"default": "small",
+			"type": "pickString",
+			"options": [
+				{ "label": "default (PR)", "value": "" },
+				{ "label": "full (CI)", "value": "FULL" },
+			],
+		},
+		{
+			"id": "verboseConsoleOutputt",
+			"description": "Show console output?",
+			"default": "",
+			"type": "pickString",
+			"options": [{ "label": "No", "value": "" }, { "label": "Yes", "value": "1" }],
+		},
+		{
+			"id": "debugOldFluidModules",
+			"description": "Load source maps for compat matrix? (slower, but allows breakpoints in compat Fluid packages)",
+			"type": "pickString",
+			"default": "!**/node_modules/**",
+			"options": [
+				{ "label": "No", "value": "!**/node_modules/**" },
+				// VSCode doesn't correctly handle an empty string in outFiles. This is an arbitrary glob that doesn't match any files.
+				{ "label": "Yes", "value": "**/dummy-folder-unused/**" },
+			],
+		},
+		{
+			"id": "odspEndpoint",
+			"description": "Which odsp endpoint should be used?",
+			"type": "pickString",
+			"default": "odsp",
+			"options": ["odsp", "odsp-df"],
+		},
```

Comment:
Just for reference, we could do something similar for r11s tests against local docker (the endpoint in that case is `docker` if I recall correctly) but it's such a rare case that I don't mind skipping it for now.

Reply:
good idea. I can't test it, but added anyway in line with comments we have in root launch.json.